topics/microbiome/tutorials/diversity/tutorial.md

```diff
@@ -0,0 +1,437 @@
+---
+layout: tutorial_hands_on
```

Comment:
```suggestion
layout: tutorial_hands_on
draft: true
```

Reply:
Do you want to have it as a draft tutorial?

Reply:
until I fix it yes

Reply:
never mind, I merged too early, lets fix it now